agemo/legacy/inverse.py

```diff
@@ -1,4 +1,4 @@
-import sage.all
+import sympy
 
 # processing generating function: inverse laplace
 def inverse_laplace(equation, dummy_variable):
```

Comment:
should we remove the original functions that use sage for the inverse?

Reply:
Yes! Thanks for spotting this.